src/libtiled/tile.cpp

```diff
@@ -41,6 +41,7 @@ Tile::Tile(int id, Tileset *tileset):
     mProbability(1.f),
     mObjectGroup(nullptr),
     mCurrentFrameIndex(0),
+    mOffset(0,0),
```

Comment:
Please leave these out since the default constructor will also create a 0,0 point.